index.js

```diff
@@ -119,4 +121,8 @@ function lookup(path, isExecutable) {
     }
 }
 
+function escapeSpaces(str) {
+    return str.replace(/ /g, '\\ ');
```

Comment:
wonder whether there's an api to escape everything needs to be escaped in path, instead of manually escaping space only